serve-api/handlers/base.go

```diff
@@ -0,0 +1,15 @@
+package handlers
+
+import "github.com/ministryofjustice/serve-opg/serve-api/entity"
+
+type BaseHandler struct {
+	orderRepo entity.OrderRepository
+}
+
+func NewBaseHandler(
+	orderRepo entity.OrderRepository,
+) *BaseHandler {
+	return &BaseHandler{
+		orderRepo: orderRepo,
```

Comment:
Repos for other entities will be created & added as per required in future tickets